### DISCUSSION TITLE
Download all recordings

### DISCUSSION THREAD

**Discussion by calyptura** [mcguirepr89/BirdNET-Pi#997] [Q&A]
Dear Birdneters,

Is there a way I can download to my computer all at once the bird sound records that are saved on my birdnet-pi? 

Thank you for any help.

Luciano

**Comment by calyptura** [comment]
(I forgot to say that my computer is a Mac M2)

**Comment by drunkenungulate** [comment]
You can use rsync for this. It can use an ssh source or destination path. So if your user is moe on both machines, and your birdnet pi server is birdnetpi.local, you could use a command like this:

`rsync -av moe@birdnetpi.local:/home/moe/BirdSongs/Extracted/ /Users/Moe/Desktop/BirdNET-Pi-Sounds`

This would copy all of the .mp3 extracted by date and common name, and save them to a BirdNET-Pi-Sounds directory on the desktop of the mac. Adjust that as needed. You'll get prompted for the password of the moe@birdnetpi.local account, just like if you werre doing a normal ssh connection.  

Oh, and one detail about rsync is that if you have a source path with a trailing '/' in it, like in my example, it will copy the contents of the directory to the destination. If you want to copy the folder itself, leave off the last slash '/' and you will get the Extracted folder itself.

**Comment by srd424** [comment]
I'm not a macOS user, but a quick google threw up https://cyberduck.io/ as a graphical sftp browser that might be of some use as well.
